type.proto

```diff
@@ -607,8 +605,41 @@ message PtObject {
     optional int32 score = 19; // DEPRECATED
     optional Trip trip = 20;
     repeated int32 scores = 21;
+    repeated EquipmentDetails equipment_details = 22;
 }
 
 message FareZone {
     optional string name = 1;   
 }
+
+message EquipmentDetails {
+    enum EquipmentType {
+        escalator = 1;
+        elevator = 2;    
+    }
+    optional string id = 1;
+    optional string name = 2;
+    optional EquipmentType embedded_type = 3;
+    optional CurrentAvailability current_availability = 4;
+}
+
+message CurrentAvailability {
+    enum EquipmentStatus {
+        unknown = 0;
+        available = 1;
+        unavailable = 2;
+    }
+    optional EquipmentStatus status = 1;
+    repeated Period periods = 2;
+    optional string updated_at = 3;
+    optional Cause cause = 4;
+    optional Effect effect = 5;
+}
+
+message Cause {
```

Comment:
(Same question for Effect)
Shouldn't we reuse the one from chaos?
This can ease its use for integration and probably for serialization maybe.
There can also be some things usable in the future in it?
But maybe there is an interface-debt on the ones from chaos...

Reply:
The message Cause form chaos-proto is different:
https://github.com/CanalTP/chaos-proto/blob/d8fbae01eef62eb65378a75ba640cfc72a87d1e5/chaos.proto#L53

Reply:
Cause from chaos is larger, yes, but equipment's use could fit in (don't know if it's smart). :question: 
For effect I remember you explicitly chose to avoid an enum, so yes, no reusing chaos' one :+1: 
Maybe just name it EquipmentCause and EquipmentEffect then to avoid misunderstanding in code? :bulb:

Reply:
It's not the same thing.